fighthealthinsurance/models.py

```
@@ -2216,6 +2216,117 @@ def __str__(self):
+
+
+@receiver(post_delete, sender=PolicyDocument)
+def _delete_policy_document_file(
+    sender: type, instance: "PolicyDocument", **kwargs: typing.Any
+) -> None:
+    """Remove the encrypted file blob when a PolicyDocument row is deleted."""
+    if instance.document_enc:
+        try:
+            instance.document_enc.delete(save=False)
+        except Exception:
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)